src/Fixer/PhpBasic/CodeStyle/DocBlockWhitespaceFixer.php

```diff
@@ -62,14 +66,15 @@ protected function applyFix(SplFileInfo $file, Tokens $tokens)
                         $matches[1] = preg_replace(
                             '/\s{2,}/',
                             ' ',
-                            $matches[1]
+                            $matches[1],
                         );
 
                         $lines[$key] = implode($matches);
                     }
                 }
 
-                $token->setContent(implode($lines));
+                $tokens[$index] = new Token([$token->getId(), implode($lines)]);
+//                $token->setContent(implode($lines));
```

Comment:
No need to leave commented out code, prev code version is available in git history

Reply:
Fixed